cmd/launcher/uninstall.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kolide/kit/logutil"
+	"github.com/peterbourgon/ff/v3"
+	"github.com/pkg/errors"
+)
+
+func runUninstall(args []string) error {
+	var (
+		flagset  = flag.NewFlagSet("kolide uninstaller", flag.ExitOnError)
+		flconfig = flagset.String(
+			"config",
+			"",
+			"launcher flags configuration file",
+		)
+	)
+
+	if err := ff.Parse(flagset, args, ff.WithEnvVarNoPrefix()); err != nil {
+		return fmt.Errorf("parsing flags: %w", err)
+	}
+
+	logger := logutil.NewCLILogger(true)
+
+	// Attempt to get the identifier from the config file if one was provided
+	identifier, err := getIdentifierFromConfigFile(*flconfig)
+	if err != nil {
+		return err
+	}
+
+	err = removeLauncher(context.Background(), logger, identifier)
+
+	return err
+}
+
+func getIdentifierFromConfigFile(configFilePath string) (string, error) {
+	var identifier string
```

Comment:
a bit of a nit pick, but I don't think there is any reason to declare the var in advance